README.md

```diff
@@ -11,81 +13,36 @@ Client library for accessing Gro Intelligence's agricultural data platform.
 
-```sh
-gro_client --user_email=email@example.com --print_token
-```
-
-This token is used throughout the Gro API client code, wherever authentication is required.
-
-Gro authentication tokens *may* expire or be reset by the administrator,
-depending on your account status. You can re-run the above command to get a new one 
-if your previous one has expired.
-
-### Saving your token as an environment variable
-
-If you don't want to enter a password or token each time, you can save
-the token as an environment variable. Please consult your OS or IDE documentation for information on how to set environment variables, e.g. [setting environment variables in Windows Powershell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-6) and [Mac OS X/Linux](https://apple.stackexchange.com/questions/106778/how-do-i-set-environment-variables-on-os-x).  In some of the sample code, it is assumed that you have the token saved to your environment variables as GROAPI_TOKEN. 
+If you don't want to enter a password or token each time, you can save the token as an environment variable. Please consult your OS or IDE documentation for information on how to set environment variables, e.g. [setting environment variables in Windows Powershell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-6) and [Mac OS X/Linux](https://apple.stackexchange.com/questions/106778/how-do-i-set-environment-variables-on-os-x). In some of the sample code, it is assumed that you have the token saved to your environment variables as `GROAPI_TOKEN`.
 
 ## Examples
 
 Navigate to [api/client/samples/](api/client/samples/) folder and try executing the provided examples.
 
-Try [quick_start.py](api/client/samples/quick_start.py)
+1. Try [quick_start.py](api/client/samples/quick_start.py).
 
 ```sh
-cd your-gro-path/api-client/api/client/samples/
-
 python quick_start.py
 ```
-Now should be able to find a sample output csv file at: `gro_client_output.csv`
 
-A more advanced example is [sugar.py](api/client/samples/crop_models/sugar.py)
+If the API client is installed and your authentication token is set, a csv file called `gro_client_output.csv` should be created in the directory where the script was run.
 
-```sh
-cd your-gro-path/api-client/api/client/samples/crop_models/
-python sugar.py
-```
-
-You can also use the Gro CLI as a quick and easy way to request a single data series right on the command line. You can either provide your email and enter a password when prompted, or you can provide your --token to avoid typing a password every time:
+2. Try out [soybeans.py](api/client/samples/crop_models/soybeans.py) to see the crop weighted series feature in action:
 
 ```sh
-gro_client --metric='Production Quantity mass' --item='Corn' --region='United States' --user_email='email@expample.com'
```

Comment:
we should keep this command line gro_client (or python gro_client.py) example as well.  since it's not udner "samples" maybe we move ti to the beginnig or the end of the examples

also lets use "command line" rather than "cli", which may be obscure for some of  our audience.

Reply:
If we add the command line back, I think more instruction should be provided behind what it's doing and why it might be helpful. I was trying to understand what this sample was doing yesterday without much luck. I believe it returns a random example? If so, I think we should use an example that is more helpful than that.

Reply:
I just added this back with some description.

I want to minimize it since it's not what we want to encourage for use in production. Added a note: "This tool is useful for simple queries, but anything more complex should be done using the Python packages."

Let me know if you think that covers it